ssz-rs/src/error.rs

```diff
@@ -45,6 +48,7 @@ impl std::error::Error for Error {}
 
 #[derive(Debug)]
 pub enum TypeError {
+    /// A type is invalid within the given bounds.
```

Comment:
you can ignore this one but this feels a bit more readable to me:
```suggestion
    /// A type is invalid for the given bounds.
```